server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java

```diff
@@ -559,25 +556,55 @@ protected Symbol normalize(String functionName, Symbol... args) {
         );
     }
 
-    public void assertHasDocValueAggregator(String functionName, List<DataType<?>> argumentTypes) {
-        var aggregationFunction = (AggregationFunction<?, ?>) nodeCtx.functions().get(
-            null,
-            functionName,
-            InputColumn.mapToInputColumns(argumentTypes),
-            SearchPath.pathWithPGCatalogAndDoc()
-        );
-        var docValueAggregator = aggregationFunction.getDocValueAggregator(
-            argumentTypes,
-            Lists2.map(argumentTypes, dataType -> mock(MappedFieldType.class)),
-            List.of()
-        );
+    public void assertHasDocValueAggregator(Signature signature,
```

Comment:
the test cases using `assertHasDocValueAggregator` could be ported to `DocValueAggregatesTest`, they seem to be doing the same testing in different ways

Reply:
currently this requires `actualReturnType` and `actualArgumentTypes` to handle cases (ex. count) where they are `V`, `E` such that invoking `signature.getArgumentDataTypes()` throws.

Reply:
> could the test cases using `assertHasDocValueAggregator` be ported to `DocValueAggregatesTest`? They seem to be doing the same testing in different ways.

Not sure - the `AggregationTestCase` is intended to write tests for aggregation functions, `DocValueAggregatesTest` is indented to test the logic that creates DocValueAggregators. These are different concerns and I'd try to avoid mixing them.